### PR TITLE
Try graphBuild in solution build

### PR DESCRIPTION
**Pull request by cartermp**
lol powershell syntax is death

https://github.com/dotnet/fsharp/issues/11281

### CODE REVIEW

**Comment by cartermp** [comment]
I haven't checked the log to make sure this is actually done - @vzarytovskii unfortunately this is one area I know pretty little about in the repo. Where in the logs would I look?

**Comment by vzarytovskii** [comment]
> I haven't checked the log to make sure this is actually done - @vzarytovskii unfortunately this is one area I know pretty little about in the repo. Where in the logs would I look?

Good question. I'd say in the binlog, but I can't really tell whether there was any change.

**Comment by vzarytovskii** [comment]
Apparently, the property is `IsGraphBuild` instead of `GraphBuild`.

**Comment by cartermp** [comment]
let's try that out!

File: eng/Build.ps1 [hunk 1]
```diff
@@ -226,6 +226,7 @@ function BuildSolution([string] $solutionName) {
         /p:QuietRestoreBinaryLog=$binaryLog `
         /p:TestTargetFrameworks=$testTargetFrameworks `
         /p:DotNetBuildFromSource=$sourceBuild `
+        /p:IsGraphBuild=$True `
```
Comment (baronfel) [inline eng/Build.ps1:229]: setting the property doesn't actually trigger the different build handling, you need to use the `/graph` flag for that instead. 
```suggestion
        /graph
```

**Comment by cartermp** [comment]
```
The term '/v:$verbosity' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
System.Management.Automation.CommandNotFoundException: The term '/v:$verbosity' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
   at System.Management.Automation.ExceptionHandlingOps.CheckActionPreference(FunctionContext funcContext, Exception exception)
   at System.Management.Automation.Interpreter.ActionCallInstruction`2.Run(InterpretedFrame frame)
   at System.Management.Automation.Interpreter.EnterTryCatchFinallyInstruction.Run(InterpretedFrame frame)
   at System.Management.Automation.Interpreter.EnterTryCatchFinallyInstruction.Run(InterpretedFrame frame)
at BuildSolution, D:\a\_work\1\s\eng\Build.ps1: line 230
at <ScriptBlock>, D:\a\_work\1\s\eng\Build.ps1: line 467
```

lolwut

**Comment by vzarytovskii** [comment]
> ```
> The term '/v:$verbosity' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
> System.Management.Automation.CommandNotFoundException: The term '/v:$verbosity' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
>    at System.Management.Automation.ExceptionHandlingOps.CheckActionPreference(FunctionContext funcContext, Exception exception)
>    at System.Management.Automation.Interpreter.ActionCallInstruction`2.Run(InterpretedFrame frame)
>    at System.Management.Automation.Interpreter.EnterTryCatchFinallyInstruction.Run(InterpretedFrame frame)
>    at System.Management.Automation.Interpreter.EnterTryCatchFinallyInstruction.Run(InterpretedFrame frame)
> at BuildSolution, D:\a\_work\1\s\eng\Build.ps1: line 230
> at <ScriptBlock>, D:\a\_work\1\s\eng\Build.ps1: line 467
> ```
> 
> lolwut

I'm not sure MSBuild cmdlet understands the graph switch. I'll look at it tomorrow.

**Comment by cartermp** [comment]
yeet

**Comment by vzarytovskii** [comment]
> yeet

The fix I made was very complex, and took me two days to figure out. It will go to my connect.

**Comment by vzarytovskii** [comment]
I understand, that it's not correct to compare CI runs, but no graph build (previous one) is faster than this one:
No graph enabled: Duration = 11:08.634
Graph enabled: Duration = 12:11.019

Also, the graph run: Static graph loaded in 0.436 seconds: 1 nodes, 0 edges

I don't think it'll make any difference for our solutions.

**Comment by cartermp** [comment]
I'll close this out given the findings!

**Pull request closed without merging**